src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -112,14 +119,14 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example to allow one single parameterless annotation on the same line
+ * Example to allow multiple annotations on the same line
```

Comment:
Please add a colon `:` after this sentence

Reply:
Done